### ISSUE TITLE
popAndPushNamed addition?

### ISSUE THREAD

**Issue by RodBr** [jonataslaw/get#93]
Any chance of adding popAndPushNamed? I know that Get.offNamed is similar, but not quite what I need.

Here's some thought behind the reason: [https://stackoverflow.com/questions/59816770/what-is-the-difference-between-pushreplacementnamed-and-popandpushnamed-in-flutt](https://stackoverflow.com/questions/59816770/what-is-the-difference-between-pushreplacementnamed-and-popandpushnamed-in-flutt)

**Comment by jonataslaw** [comment]
Try Get.offAndToNamed on last version

**Comment by RodBr** [comment]
Thanks again.

Just an FYI, I created my app using the ‘Filledstacks” Stacked framework.
Today I replaced, all of the Get_It  locator code with Get.put, all of the Dialog Manager code with Get, and all of the Navigation Service code with Get.

There are a lot fewer lines of code, and it’s far less confusing to read, at least for me.



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
